x-pack/legacy/plugins/infra/public/pages/logs/analysis/sections/anomalies/index.tsx

```diff
@@ -0,0 +1,211 @@
+        </EuiFlexGroup>
+      ) : !results || (results && results.histogramBuckets && !results.histogramBuckets.length) ? (
+        <EuiEmptyPrompt
+          title={
+            <h2>
+              {i18n.translate('xpack.infra.logs.analysis.anomalySectionNoDataTitle', {
+                defaultMessage: 'There is no data to display.',
+              })}
+            </h2>
+          }
+          titleSize="m"
+          body={
+            <p>
+              {i18n.translate('xpack.infra.logs.analysis.anomalySectionNoDataBody', {
+                defaultMessage: 'You may want to adjust your time range.',
+              })}
+            </p>
+          }
+        />
+      ) : !hasAnomalies ? (
+        <EuiEmptyPrompt
+          title={
+            <h2>
+              {i18n.translate('xpack.infra.logs.analysis.anomalySectionNoAnomaliesTitle', {
+                defaultMessage: 'No anomalies were detected.',
+              })}
+            </h2>
+          }
+          titleSize="m"
+        />
+      ) : (
+        <>
+          <EuiFlexGroup>
+            <EuiFlexItem grow={8}>
+              <AnomaliesChart
+                chartId="overall"
+                setTimeRange={setTimeRange}
+                timeRange={timeRange}
+                series={logEntryRateSeries}
+                annotations={anomalyAnnotations}
+                renderAnnotationTooltip={renderAnnotationTooltip}
+              />
+            </EuiFlexItem>
+            <EuiFlexItem grow={2}>
+              <EuiStat
+                title={numeral(results.totalNumberOfLogs).format('0.00a')}
+                description={i18n.translate(
+                  'xpack.infra.logs.analysis.overallAnomaliesNumberOfLogsDescription',
+                  {
+                    defaultMessage: 'Number of logs',
+                  }
+                )}
+                reverse
+              />
+              <EuiStat
+                title={Number(topAnomalyScore).toFixed(0)}
+                description={i18n.translate(
+                  'xpack.infra.logs.analysis.overallAnomaliesTopAnomalyScoreDescription',
+                  {
+                    defaultMessage: 'Max anomaly score',
+                  }
+                )}
+                reverse
+              />
+            </EuiFlexItem>
+          </EuiFlexGroup>
+          <EuiSpacer size="l" />
+          <AnomaliesTable results={results} setTimeRange={setTimeRange} timeRange={timeRange} />
+        </>
+      )}
+    </>
+  );
+};
+
+interface ParsedAnnotationDetails {
+  anomalyScoresByPartition: Array<{ partitionId: string; maximumAnomalyScore: number }>;
+}
+
+const overallAnomalyScoreLabel = i18n.translate(
+  'xpack.infra.logs.analysis.overallAnomalyChartMaxScoresLabel',
+  {
+    defaultMessage: 'Max anomaly scores:',
+  }
+);
+const AnnotationTooltip: React.FunctionComponent<{ details: string }> = ({ details }) => {
+  const parsedDetails: ParsedAnnotationDetails = JSON.parse(details);
+  return (
+    <div>
```

Comment:
Maybe we could set `white-space: nowrap` on this to avoid the following?

![grafik](https://user-images.githubusercontent.com/973741/66779335-2866a500-eece-11e9-9004-c44de4902abe.png)

Reply:
And how about making the tooltip header bold too?